jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -351,12 +351,22 @@ private static void arraycopy(Object[] A1, int offset1, Object[] A2, int offset2
 private static final int FileEncoding = 2;
 private static final int OSEncoding = 3;
 
+/*[IF OpenJ9-RawBuild]*/
+// JCL native
```

Comment:
Can you describe here why this is only needed for the raw builds?  It will save someone from having to figure out if we can delete it in the future.

Reply:
Added comments describing the reason to add/invoke this `JCL` native for `OpenJ9` raw build and the condition to remove this workaround.